recipes-ni/ni-safemode-utils/ni-safemode-utils.bb

```diff
@@ -20,7 +20,9 @@ FILES:${PN} += "\
 
 DEPENDS += "shadow-native pseudo-native niacctbase update-rc.d-native"
 
-RDEPENDS:${PN} += "niacctbase bash fw-printenv"
+RDEPENDS:${PN} += "niacctbase bash"
+RDEPENDS:${PN}:append:x64 = " fw-printenv"
+RDEPENDS:${PN}:append:xilinx-zynq = " u-boot-fw-utils"
```

Comment:
What is the difference between `fw-printenv` and `u-boot-fw-utils`? Do they both provide the `fw-printenv` utility? If so, maybe the it would be better for us to say that `u-boot-fw-utils` RPROVIDES `fw-printenv` and that `fw-printenv` is an x64-only package.